cartographer_ros/cartographer_ros/node.cc

```diff
@@ -122,7 +122,7 @@ Node::Node(
       &Node::PublishConstraintList, this));
 }
 
-Node::~Node() {}
+Node::~Node() { FinishAllTrajectories(); }
```

Comment:
As I explained in a review comment above:

> This could be added before returning in if (!ros::ok()), but I thought it would be more appropriate to add it here.
> Without this, if you do not finish all your trajectories, you get a failed check when exiting using Ctrl-C.

Reply:
Also ptal at my answer to @cshuet: https://github.com/googlecartographer/cartographer_ros/pull/636/files#r163297172